src/schema/v2/homeView/sections/ExploreByCategories.ts

```diff
@@ -0,0 +1,64 @@
+import { ContextModule } from "@artsy/cohesion"
+import { HomeViewSection } from "."
+import { HomeViewSectionTypeNames } from "../HomeViewSection"
+
+export const ExploreByCategories: HomeViewSection = {
+  id: "home-view-section-explore-by-categories",
+  type: HomeViewSectionTypeNames.HomeViewSectionMarketingCollectionCategories,
+  // TODO: add right tracking
+  contextModule: ContextModule.categoryRail,
+  component: {
+    title: "Explore by categories",
+  },
+  requiresAuthentication: false,
+
+  resolver: () => categories,
+}
+
+// TODO: temporary images, we still need to wait for the assets and to incorporate the image versions too
+const categories = [
+  {
+    name: "Medium",
+    image: {
+      image_url: "https://files.artsy.net/images/capivara_chimarrao.jpg",
+    },
+    href: "/collection-hub/Medium",
```

Comment:
**question (non-blocking):** I don't recognize these `/collection-hub/foo` urls — is there a plan to introduce them? (This may all be very preliminary/TODO but these don't _sound_ like names of "collection hubs" to me in the sense that we have previously used that term)

Reply:
yes, it will become a new screen, so for now(if we don't release the home view in the next 2 weeks) it will give us a 404. re naming, yes I don't think it's a good name, I will change it to `marketing-collections/:category` if doesn't exist yet(I don't think it already exists but I'll double-check)